administrator/components/com_patchtester/script.php

```diff
@@ -48,6 +51,24 @@ public function __construct()
 	 */
 	public function postflight($type, $parent)
 	{
-		$this->removeFiles();
+		if ($type == 'install')
+		{
+			$language = JFactory::getLanguage();
+			$language->load('com_patchtester', JPATH_ADMINISTRATOR, null, true);
+			$language->load('com_patchtester', JPATH_SITE, null, true);
+			echo '<h1><img src="../media/com_patchtester/images/icon-48-patchtester.png"> ' . Text::_('COM_PATCHTESTER') . '</h1>';
+			echo '<p>' . Text::_('COM_PATCHTESTER_XML_DESCRIPTION') . '</p>';
+			echo '<p>' . Text::_('COM_PATCHTESTER_GOTO') . ' <a href="index.php?option=com_patchtester">' . Text::_('COM_PATCHTESTER') . '</a></p>';
+			echo '<p>' . Text::_('COM_PATCHTESTER_GOTO') . ' <a href="index.php?option=com_config&view=component&component=com_patchtester#authentication	&path=&return=aHR0cDovL2xvY2FsaG9zdC9wYXRjaDRhMTIvYWRtaW5pc3RyYXRvci9pbmRleC5waHA%2Fb3B0aW9uPWNvbV9wYXRjaHRlc3Rlcg%3D%3D">' . Text::_('COM_PATCHTESTER_OPTIONS') . '</a></p>';
+		}
+
+		if ($type == 'uninstall')
+		{
+			$language = JFactory::getLanguage();
```

Comment:
Same as above, do we need to load the language?

Reply:
I'm not to sure we need the uninstall option in there but left it in as an option

Reply:
We can test here as well if we need to load the language.